ios/Podfile

```diff
@@ -1,14 +1,15 @@
 require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 require_relative '../node_modules/react-native/scripts/react_native_pods'
 
-platform :ios, '12.4'
+platform :ios, '13.4'
 
 project 'PowerAuth'
 
 target 'PowerAuth' do
   config = use_native_modules!
   use_react_native!(
-    :path => config[:reactNativePath]
+    :path => config[:reactNativePath],
+    :hermes_enabled => false
```

Comment:
Could you describe the source of the problems? I was using "testapp" project only, for the development workflows. If I remember this correctly, then opening the standalone project was pain in the ass in recent RN stack, for both platforms. Using the "testapp" project was much better.

Reply:
Hermes is a Facebook-made javascript engine tailored for react native and brings faster startup times.

Basically, for a final build (when compiled), it improves performance but also brings many dependencies that need to be maintained and fixed when broken.

Since we use this only for our test projects and test apps, it's not needed.

Test projects for iOS and the test app still work but with the default js engine, not the Hermes.